Jenkinsfile

```diff
@@ -2,7 +2,7 @@ pipeline {
   agent any
   triggers{
     pollSCM '* * * * *'
-    cron("H H(2-5) * * *")
+    cron(BRANCH_NAME == "master" ? "H H(2-5) * * *": "")
```

Comment:
Could you also remove the `pollSCM` line. It's not required, since we use the Github jenkins plugin.

Reply:
Good point. I also removed some obsolete code and made dev the default pypi index.